db/dataset.py

```diff
@@ -89,6 +89,9 @@ def create_from_dict(dictionary, author_id):
                            (cls["name"], cls["description"], dataset_id))
             cls_id = result.fetchone()[0]
 
+            # Removing duplicate recordings
+            cls["recordings"] = list(set(cls["recordings"]))
```

Comment:
This is the same recording id multiple times in the same class, right?  Does the UI also block this?